src/main/java/com/redhat/devtools/intellij/qute/psi/core/command/QuteAction.java

```diff
@@ -13,14 +13,14 @@
 import com.google.gson.JsonPrimitive;
 import com.intellij.openapi.actionSystem.AnAction;
 import com.intellij.openapi.actionSystem.AnActionEvent;
-import com.redhat.devtools.intellij.lsp4ij.operations.codelens.LSPCodelensInlayProvider;
+import com.redhat.devtools.intellij.lsp4ij.command.internal.CommandExecutor;
```

Comment:
Move to  `com.redhat.devtools.intellij.lsp4ij.command.CommandExecutor` (it is nt an internal class since you can use it outside LSP4IJ)